deploy-tool/main.go

```diff
@@ -162,6 +162,13 @@ func main() {
 		log.Fatalf("failed to parse config.toml: %v", err)
 	}
 
+	if deployConfig.Job.Enable {
+		handleJob(deployConfig)
+		// TODO eventually maybe use the output from this to generalize pushing the changes with gsutil
```

Comment:
Well now that this tool has a proper maintainer and it's a real Go module with deps and everything, it should properly pull in the GCP SDK and manage bucket contents that way. Same goes for those ghastly `kubectl` invocations; they should be replaced with proper API usage--and then we wouldn't have to write out keys to temp files.